interfaces/builtin/fwupd.go

```diff
@@ -148,6 +157,20 @@ const fwupdConnectedPlugAppArmor = `
       interface=org.freedesktop.DBus.Properties
       peer=(label=###SLOT_SECURITY_TAGS###),
 
+  dbus (send)
+      bus=system
+      path=/org/freedesktop/systemd1
+      interface=org.freedesktop.systemd1.Manager
+      member={GetDefaultTarget,GetUnit}
+      peer=(label=unconfined),
```

Comment:
These Get APIs are 'ok' for fwupd, but are getting dangerously close to systemctl service management. Be warned, we won't be allowing access to the systemctl command for managing services in this or any other interface. If that is the intended future direction, please raise this with @mvo5 and @pedronis to discuss how fwupd and snapd should be adjusted for arbitrary service management.

Reply:
I suspect it won't actually impact firmware update but just fwupd gave the apparmor denied messages during updating. I will confirm against this case.